packages/tools/src/stateManagement/annotation/config/getState.ts

```diff
@@ -22,6 +22,10 @@ function getState(annotation?: Annotation): AnnotationStyleStates {
     if (isAnnotationLocked(annotation)) {
       return AnnotationStyleStates.Locked;
     }
+
+    if (annotation.data && annotation.autoGenerated) {
```

Comment:
The positioning of this says what the state diagram is - but I think you need to be specific about it up above, as it wasn't clear without getting this far into the weeds.